db/re/skill_db.yml

```
@@ -45018,6 +45018,49 @@ Body:
         - Level: 5
           Amount: 78
     Status: WeaponBlock_On
+  - Id: 6520
+    Name: WH_WILD_WALK
+    Description: Wild Walk
+    MaxLevel: 5
+    Type: Weapon
+    TargetType: Attack
+    DamageFlags:
+      Critical: true
+    Flags:
+      AllowOnWarg: true
+    Range: 11
+    Hit: Multi_Hit
+    HitCount: -3
+    CastCancel: true
+    FixedCastTime: 50
```

```suggestion
    CastTime: 3000
    FixedCastTime: 350
    AfterCastActDelay: 500
```
at level 5
(cast 768 nude, 418 with fixcast -0.5s ; 148 dex, 99 int)